### PR TITLE
Ic sheet12 boosting + cleanup

### PR DESCRIPTION
**Pull request by patriciomassaro**
- Clean up of the folder ( descriptive names, changing the paths in the exercise sheets, etc)
- Updating collection
- Adding solutions to IC exercise sheet

### CODE REVIEW

**Comment by patriciomassaro** [comment]
[IC exercise SL - Sheet 12](https://trello.com/c/R5EsVkZ1/262-ic-exercise-sl-sheet-12)

**Review by ludwigbothmann — COMMENTED** [review]
@patriciomassaro Why do we have an ex_boosting.Rnw and an ex_boosting_1.Rnw? I guess these are the 2 exercise sheets. If so: please name them `1` and `2`. And where is the solution of the second sheet?

File: exercises/boosting/ic_boosting_1.Rnw [hunk 1]
```diff
@@ -15,18 +15,18 @@ knitr::set_parent("../../style/preamble_ueb.Rnw")
...
 \aufgabe{AdaBoost - Decision Stump}{
-	<<child="ex_rnw/ic_boosting_2.Rnw">>=
+	<<child="ex_rnw/ic_boosting_stumps.Rnw">>=
 	@
 }
 \lz
 
 \aufgabe{Quiztime}{
-	<<child="ex_rnw/ic_boosting_3.Rnw">>=
+	<<child="ex_rnw/ex_particify_1.Rnw">>=
```
Comment (ludwigbothmann) [inline exercises/boosting/ic_boosting_1.Rnw:30]: please remove the quiz

**Review by ludwigbothmann — COMMENTED** [review]
Apart of this, the solution looks very thorough, thanks @patriciomassaro !

**Comment by patriciomassaro** [comment]
> @patriciomassaro Why do we have an ex_boosting.Rnw and an ex_boosting_1.Rnw? I guess these are the 2 exercise sheets. If so: please name them `1` and `2`. And where is the solution of the second sheet?

Thanks for noting that Ludwig. the "ex_boosting.Rnw" file points to an exam exercise ( see attachment) and I don't know if it should be part of the exercise sheets. Our options are: 

- I can rename it to "ex_boosting_2.Rnw" and add it's solution too as sol_boosting_2.Rnw". 
- I can delete "ex_boosting.Rnw" and keep the exam exercise only in the collection.
image

File: exercises/boosting/ic_boosting_1.Rnw (hunk 1 above)
Comment (patriciomassaro) [reply exercises/boosting/ic_boosting_1.Rnw]: Done

**Comment by patriciomassaro** [comment]
> 

Done :)

**Comment by patriciomassaro** [comment]
> @patriciomassaro Why do we have an ex_boosting.Rnw and an ex_boosting_1.Rnw? I guess these are the 2 exercise sheets. If so: please name them `1` and `2`. And where is the solution of the second sheet?

Done 👍

**Review by ludwigbothmann — APPROVED** [review]

**Pull request merged**